Updates the council section (#780)
w3f/polkadot-wiki: docs/learn-governance.md

## Council

To represent passive stakeholders, Polkadot introduces the idea of a "council". The council is an

> on-chain entity comprising a number of actors, each represented as an on-chain account. On Polkadot

Reviewer: ```suggestion
on-chain entity comprising a number of actors, each represented as an on-chain account. On Polkadot,
```

> the council currently consists of 13 members, however this is expected to increase over the next few

Reviewer: ```suggestion
the council currently consists of 13 members. This is expected to increase over the next few
```

> months to 24 people. In general, the council will end up having a fixed number of seats. On

Reviewer: ```suggestion
months to 24 seats. In general, the council will end up having a fixed number of seats. On
```

Reviewer: The "in general" sentence is superfluous I think. Also recommend changing people to seats, since there's already people who hold more than one seat.

> Polkadot, this will be 24 seats while on Kusama it is 17 seats.

Reviewer: ```suggestion
Polkadot, this will be 24 seats while on Kusama it is 17 seats.
```

Won't they both end up at 24? Why not just say that the number of seats will gradually increase until both councils reach 24?

Author: Is it true for Kusama too? Polkadot's governance has always had the vision for 24 council seats but I think Kusama might branch off here and remain at 17.

The council is called upon primarily for three tasks of governance: proposing sensible referenda,
cancelling uncontroversially dangerous or malicious referenda, and electing the technical committee.

For a referendum to be proposed by the council, a strict majority of members must be in favor, with
no member exercising a veto. Vetoes may be exercised only once by a member for any single proposal;
if, after a cool-down period, the proposal is resubmitted, they may not veto it a second time.

> Council motions which pass with a strict majority - but without reaching unanimous support will move

Reviewer: ```suggestion
Council motions which pass with a strict majority - but without reaching unanimous support - will move
```

> to public referendum under a neutral, majority-carries adaptive quorum biasing. In the case that all

Reviewer: ```suggestion
to public referendum under a neutral, majority-carries voting scheme. In the case that all
```

AQB is only when we need supermajority relative to turnout, no?

Author: Yes. I was using AQB here as the catch-all for however the voting scheme ends up. Strictly speaking, it only does apply when the supermajority is required relative to turnout.

![](assets/governance/approval-vote.png)

> Currently there is 13 seats on the council. All stakeholders are free to signal their approval of

Reviewer: ```suggestion
Currently there are 13 seats on the council. All stakeholders are free to signal their approval of
```

any of the registered candidates.

Council elections are handled by the same [Phragmén election](learn-phragmen) process that selects

> validators from the available pool based on nominations. However, token holders' vote for

Reviewer: ```suggestion
validators from the available pool based on nominations. However, token holders' votes for
```

councillors are isolated from any of the nominations they may have on validators. Council terms last

> for one day. At the end of each term, Phragmen election algorithm is ran and the result will choose

Reviewer: ```suggestion
for one day. At the end of each term, Phragmen election algorithm runs and the result will choose
```
